code/controllers/Processes/shuttles.dm

```diff
@@ -84,7 +88,7 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 	for(var/obj/docking_port/stationary/S in stationary)
 		if(S.id == id)
 			return S
-	WARNING("couldn't find dock with id: [id]")
+	//WARNING("couldn't find dock with id: [id]")
```

Comment:
This should probably leave at least a debugging message or something so that it doesn't fail silently when an admin is working with this, or something